jvm-packages/xgboost4j-spark-gpu/src/main/scala/ml/dmlc/xgboost4j/scala/spark/GpuXGBoostPlugin.scala

```diff
@@ -170,6 +186,20 @@ class GpuXGBoostPlugin extends XGBoostPlugin {
         }
       }
     )
+
+    val sconf = dataset.sparkSession.conf
+    val rmmEnabled: Boolean = try {
+      sconf.get("spark.rapids.memory.gpu.pooling.enabled").toBoolean &&
```

Comment:
We need to have a brief note for these configurations in the xgb spark document, with links to spark rapids https://nvidia.github.io/spark-rapids/docs/additional-functionality/advanced_configs.html

Reply:
I added the document.